ui/lib/utils/i18n.ts

```diff
@@ -60,4 +60,8 @@ i18next
     },
   })
 
+export function isDistro() {
+  return process.env.REACT_APP_DISTRO_BUILD_TAG !== undefined
```

Comment:
Tried use `return process.env.REACT_APP_DISTRO_BUILD_TAG !== 1` to judge, but can't pass the ts compile, it says process.env.xxx values are string | null，although in this case, process.env.REACT_APP_DISTRO_BUILD_TAG is a number.

Reply:
Maybe we can export the variables directly, like `const isDistro = process.env.REACT_APP_DISTRO_BUILD_TAG !== undefined`.

Reply:
Make sense, let me udpate it.